packages/xml-views-quick-fix/api.d.ts

```diff
@@ -0,0 +1,8 @@
+import { XMLDocument } from "@xml-tools/ast";
+import { OffsetRange } from "@ui5-language-assistant/logic-utils";
```

Comment:
if you have a dependency to logic-utils in the api.d.ts
it also has to be in dependency section in package.json (not devDep)